From 02470fc372b023730b3e7ef9f660c17945d2b4f4 Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Fri, 1 Jun 2018 22:50:07 -0700
Subject: [PATCH 3/5] LineageParts: Use the battery HAL lights brightness
 capability

*) https://review.lineageos.org/#/c/LineageOS/android_lineage-sdk/+/216474/
   introduces capability LIGHTS_ADJUSTABLE_BATTERY_LED_BRIGHTNESS to
   allow a device to indicate whether it's liblights supports battery light
   brightness control.  Update lineage lights settings to use this new
   capability.

Change-Id: If51a609b0ca88aaf5163003e8e1a2696d8013580
---
 .../lineageparts/notificationlight/BatteryLightSettings.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/org/lineageos/lineageparts/notificationlight/BatteryLightSettings.java b/src/org/lineageos/lineageparts/notificationlight/BatteryLightSettings.java
index c723156..dc640e1 100644
--- a/src/org/lineageos/lineageparts/notificationlight/BatteryLightSettings.java
+++ b/src/org/lineageos/lineageparts/notificationlight/BatteryLightSettings.java
@@ -79,7 +79,7 @@ public class BatteryLightSettings extends SettingsPreferenceFragment implements
         mMultiColorLed =
                 LightsCapabilities.supports(context, LightsCapabilities.LIGHTS_RGB_BATTERY_LED);
         mHALAdjustableBrightness = LightsCapabilities.supports(
-                context, LightsCapabilities.LIGHTS_ADJUSTABLE_NOTIFICATION_LED_BRIGHTNESS);
+                context, LightsCapabilities.LIGHTS_ADJUSTABLE_BATTERY_LED_BRIGHTNESS);
         final boolean pulsatingLed =
                 LightsCapabilities.supports(context, LightsCapabilities.LIGHTS_PULSATING_LED);
         final boolean segmentedBatteryLed =
-- 
2.17.0

